.rhcicd/pr_check.sh

```diff
@@ -36,3 +36,13 @@ cat << EOF > ${WORKSPACE}/artifacts/junit-dummy.xml
     <testcase classname="dummy" name="dummytest"/>
 </testsuite>
 EOF
+
+# adding floorist tests.
```

Comment:
Consider replacing the above block that is dummying the artifact file, since you don't need it when your tests are actually added and running. 

It's a placeholder for when the CJI isn't running anything yet.